zbarimg/zbarimg.c

```diff
@@ -167,6 +167,13 @@ static int scan_image(const char *filename)
 
     int found	       = 0;
     MagickWand *images = NewMagickWand();
+    if (!MagickPingImage(images, filename) && dump_error(images))
+    return (-1);
```

Comment:
Indentation

Reply:
Help me out here, is there a standard tab/space conversion?

Reply:
Well this file is using a somewhat random mix of tabs and spaces, looks to be 4 spaces per tab, can probably just use tabs here.